modules/concurrency/include/hephaestus/concurrency/context.h

```diff
@@ -0,0 +1,72 @@
+//=================================================================================================
```

Comment:
consider moving this into a `context` subfolder and adding `context` namespace (to match the folder structure)